### PR TITLE
apps/example/testcase: Fix warning messages

### PR DESCRIPTION
**Pull request by sangam-swami**

### CODE REVIEW

File: apps/examples/testcase/le_tc/drivers/tc_bch.c [hunk 1]
```diff
@@ -40,14 +40,17 @@ static inline void close_fds(int *fds, int count)
 /* Close given file descriptor and free the buffer */
 #define cleanup(fd, buf) \
 do {\
-	close(fd);\
-	free(buf);\
+	if(fd > 0)\
```
Comment (jeongchanKim) [inline apps/examples/testcase/le_tc/drivers/tc_bch.c:43]: Please add space after `if`.

File: apps/examples/testcase/le_tc/drivers/tc_bch.c [hunk 2]
```diff
@@ -40,14 +40,17 @@ static inline void close_fds(int *fds, int count)
...
+		close(fd);\
+	if(buf)\
```
Comment (jeongchanKim) [inline apps/examples/testcase/le_tc/drivers/tc_bch.c:45]: Same as above.

File: apps/examples/testcase/le_tc/drivers/tc_bch.c [hunk 3]
```diff
@@ -40,14 +40,17 @@ static inline void close_fds(int *fds, int count)
...
+		free(buf);\
 } while (0)
 
 /* Close the file descriptor and unregister the bch device */
 #define clean_unreg(fd) \
 do {\
-	close(fd);\
+	if(fd > 0)\
```
Comment (jeongchanKim) [inline apps/examples/testcase/le_tc/drivers/tc_bch.c:52]: Same as above.

File: apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c [hunk 4]
```diff
@@ -886,16 +886,16 @@ static void itc_wifimanager_success_ratio_ap(void)
 
 	printf("\nSuccess Ratio \n");
 	if (init_cnt > 0) {
-		printf("[Initialization Wifi] Success Ratio = %.2f% \n", (s_init_cnt / init_cnt) * 100);
+		printf("[Initialization Wifi] Success Ratio = %.2f \n", (s_init_cnt / init_cnt) * 100);
```
Comment (jeongchanKim) [inline apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c:889]: I think original `%` wants to represent the percentage, not the format specifier.
So you should modify to `%%`, not removing.

File: apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c [hunk 5]
```diff
@@ -886,16 +886,16 @@ static void itc_wifimanager_success_ratio_ap(void)
...
-		printf("[Connect Wifi] Success Ratio = %.2f% \n", (s_join_cnt / join_cnt) * 100);
+		printf("[Connect Wifi] Success Ratio = %.2f \n", (s_join_cnt / join_cnt) * 100);
 	}
 	if (leave_cnt > 0) {
-		printf("[Disconnect Wifi] Success Ratio = %.2f% \n", (s_leave_cnt / leave_cnt) * 100);
+		printf("[Disconnect Wifi] Success Ratio = %.2f \n", (s_leave_cnt / leave_cnt) * 100);
 	}
 	if (deinit_cnt > 0) {
-		printf("[Deinitization Wifi] Success Ratio = %.2f% \n", (s_deinit_cnt / deinit_cnt) * 100);
+		printf("[Deinitization Wifi] Success Ratio = %.2f \n", (s_deinit_cnt / deinit_cnt) * 100);
```
Comment (jeongchanKim) [inline apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c:898]: I know this is not related to what you want to change, but I think `Deinitization` is not the correct word.
How about just `Deinit`?

**Review by jeongchanKim — APPROVED** [review]
LGTM

File: apps/examples/testcase/le_tc/drivers/tc_bch.c (hunk 1 above)
Comment (sangam-swami) [reply apps/examples/testcase/le_tc/drivers/tc_bch.c]: done

File: apps/examples/testcase/le_tc/drivers/tc_bch.c (hunk 2 above)
Comment (sangam-swami) [reply apps/examples/testcase/le_tc/drivers/tc_bch.c]: done

File: apps/examples/testcase/le_tc/drivers/tc_bch.c (hunk 3 above)
Comment (sangam-swami) [reply apps/examples/testcase/le_tc/drivers/tc_bch.c]: done

File: apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c (hunk 4 above)
Comment (sangam-swami) [reply apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c]: yes, changed for percentage display

File: apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c (hunk 5 above)
Comment (sangam-swami) [reply apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c]: updated

File: apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c [hunk 6]
```diff
@@ -886,16 +886,16 @@ static void itc_wifimanager_success_ratio_ap(void)
...
-		printf("[Connect Wifi] Success Ratio = %.2f% \n", (s_join_cnt / join_cnt) * 100);
+		printf("[Connect Wifi] Success Ratio = %.2f%% \n", (s_join_cnt / join_cnt) * 100);
 	}
 	if (leave_cnt > 0) {
-		printf("[Disconnect Wifi] Success Ratio = %.2f% \n", (s_leave_cnt / leave_cnt) * 100);
+		printf("[Disconnect Wifi] Success Ratio = %.2f%% \n", (s_leave_cnt / leave_cnt) * 100);
 	}
 	if (deinit_cnt > 0) {
-		printf("[Deinitization Wifi] Success Ratio = %.2f% \n", (s_deinit_cnt / deinit_cnt) * 100);
+		printf("[Deinit Wifi] Success Ratio = %.2f%% \n", (s_deinit_cnt / deinit_cnt) * 100);
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c:898]: Other messages have full word. How about `Deinitialization` here?

**Review by jeongchanKim — DISMISSED** [review]
LGTM

File: apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c (hunk 6 above)
Comment (sangam-swami) [reply apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifimanager_main.c]: done

**Comment by deepaksrma** [comment]
LGTM

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**